geometry_msgs/Point32 goal
---
bool success